src/pages/Admin/Venue/AnnouncementOptions/AnnouoncementStatus/AnnouncementStatus.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import classNames from "classnames";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCheck, faTimes } from "@fortawesome/free-solid-svg-icons";
+
+import { Banner } from "types/banner";
+
+import { isDefined } from "utils/types";
+
+import "./AnnouncementStatus.scss";
+
+export interface AnnouncementStatusProps {
+  banner?: Banner;
+}
+
+const ENABLED__TEXT = "enabled";
+const DISABLED__TEXT = "disabled";
+
+export const AnnouncementStatus: React.FC<AnnouncementStatusProps> = ({
+  banner,
+}) => {
+  const fullscreenImg = banner?.isFullScreen ? faCheck : faTimes;
+  const closeImg = banner?.hasCloseButton ? faCheck : faTimes;
+
+  const fullscreenText = banner?.isFullScreen ? ENABLED__TEXT : DISABLED__TEXT;
+  const forceText = banner?.hasCloseButton ? ENABLED__TEXT : DISABLED__TEXT;
+
+  const urlText = banner?.isActionButton ? banner?.buttonUrl : "blank";
+
+  const urlTextClassNames = classNames("AnnouncementStatus__url-text", {
```

Comment:
We use `fooClasses` for the name of these variables:
```suggestion
  const urlTextClasses = classNames("AnnouncementStatus__url-text", {
```

Reply:
done 771b70a